go_lib/deckhouse-config/kube_helpers.go

```diff
@@ -99,3 +103,54 @@ func GetAllConfigs(kubeClient k8s.Client) ([]*d8cfg_v1alpha1.ModuleConfig, error
 
 	return objs, nil
 }
+
+// SetModuleConfigEnabledFlag updates spec.enabled flag or creates a new ModuleConfig with spec.enabled flag.
+func SetModuleConfigEnabledFlag(kubeClient k8s.Client, name string, enabled bool) error {
+	// This should not happen, because DebugServer is started after Kubernetes client initialization, but check it anyway.
+	if kubeClient == nil {
+		return fmt.Errorf("kubernetes client not initialized, try again later")
+	}
+
+	gvr := d8cfg_v1alpha1.GroupVersionResource()
+	unstructuredObj, err := kubeClient.Dynamic().Resource(gvr).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil && !k8errors.IsNotFound(err) {
+		return err
```

Comment:
I suggest wrapping errors with a description: 

```go
return fmt.Errorf("getting module config: %w", err)
return fmt.Errorf("updating module config: %w", err)
return fmt.Errorf("creating module config: %w", err)
```

In this function, with kube client we 1) get, 2) update, or 3) create an object. If we have any non-404 issue, we should just get an error from the API. I'd prefer not to loose track of a specific action causing the problem.

Reply:
Done